packages/google_maps_flutter/lib/src/ui.dart

```diff
@@ -125,6 +126,18 @@ class GoogleMapOptions {
   /// True if the map view should respond to zoom gestures.
   final bool zoomGesturesEnabled;
 
+  /// Whether to show a "My Location" layer over the map.
+  ///
+  /// The layer includes a location indicator and My Location button at the current device location if it is known.
+  /// The indicator is a small blue dot if the device is stationary, or a chevron if the device is moving.
+  /// The My Location button animates to focus on the user's current location if the user's location is currently known.
+  ///
+  /// Enabling this feature requires adding location permissions.
+  /// On Android add either `<uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />` or `<uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />` to your `AndroidManifest.xml` file.
```

Comment:
Did you mean for one of these to be coarse location?
Also probably worth including enough information for someone reading this to figure which permission he should add.